Update directory listing of release-build structure (#8010)
flutter/website: src/deployment/web.md

      MaterialIcons-Regular.ttf
      <other font files>
    <image files>
    packages
      cupertino_icons
        assets
          CupertinoIcons.ttf
    shaders
      ink_sparkle.frag
  canvaskit
    canvaskit.js
    canvaskit.wasm
    profiling
      canvaskit.js
      canvaskit.wasm
  favicon.png
  flutter.js
  flutter_service_worker.js
  index.html
  main.dart.js
  manifest.json
  version.json
```

> {{site.alert.note}}
> The `canvaskit` directory and its contents will only be present when the
> CanvasKit renderer is selected — but not when the HTML renderer is selected.
> {{site.alert.end}}
> 

Reviewer: ```suggestion
{{site.alert.note}}
  The `canvaskit` directory and its contents are only present when the
  CanvasKit renderer is selected—not when the HTML renderer is selected.
{{site.alert.end}}

```
